launcher/src/main.rs

```diff
@@ -72,17 +82,17 @@ enum Quantization {
     Marlin,
     /// Bitsandbytes 8bit. Can be applied on any model, will cut the memory requirement in half,
     /// but it is known that the model will be much slower to run than the native f16.
-    #[deprecated(
-        since = "1.1.0",
-        note = "Use `eetq` instead, which provides better latencies overall and is drop-in in most cases"
-    )]
+    // #[deprecated(
+    //     since = "1.1.0",
+    //     note = "Use `eetq` instead, which provides better latencies overall and is drop-in in most cases"
+    // )]
```

Comment:
We cna't keep that easily because `Deserialize` macro will use that field, and there's  no way for us to insert `#[allow(deprecated)]` in the generated code.

Reply:
Would it be possible to add the deprecation in the Display impl instead? I think only Clap uses Display.